js_modules/dagit/packages/core/src/workspace/WorkspaceTabs.tsx

```diff
@@ -18,8 +18,8 @@ export const WorkspaceTabs = <TData extends Record<string, any>>(props: Props<TD
     <Box flex={{direction: 'row', justifyContent: 'space-between', alignItems: 'flex-end'}}>
       <Tabs selectedTabId={tab}>
         <TabLink id="jobs" title="Jobs" to="/workspace/jobs" />
-        {/* <TabLink id="schedules" title="Schedules" to="/workspace/schedules" />
-        <TabLink id="sensors" title="Sensors" to="/workspace/sensors" />
+        <TabLink id="schedules" title="Schedules" to="/workspace/schedules" />
+        {/* <TabLink id="sensors" title="Sensors" to="/workspace/sensors" />
```

Comment:
should this all be visible?

Reply:
I'm exposing them in the tabs as I add the routes. The feature is flagged anyway, so no one will see this page unless they turn the flag on, and the routes referenced in the commented tabs don't exist yet.